permits/tests/test_permit_request_api.py

```diff
@@ -463,6 +463,22 @@ def test_authorized_network_does_not_raise_exception(self):
         )
         self.assertEqual(response.status_code, 200)
 
+    def test_search_api_found(self):
+        self.client.login(username=self.admin_user.username, password="password")
+        author_permit_request = models.PermitRequest.objects.first().author
+        response = self.client.get(
+            reverse("search-list"), {"search": author_permit_request}
+        )
+        response_json = response.json()
+        self.assertEqual(response.status_code, 200)
+        self.assertNotEqual(response_json, [])
+
+    def test_search_api_nothing_found(self):
+        self.client.login(username=self.admin_user.username, password="password")
+        response = self.client.get(
+            reverse("search-list"), {"search": "InexistantStringReturningNoResult"}
+        )
+
```

Comment:
Add test to check that unauthentified/unauthorized user can't get results for permits he has no right to see